library/ssl_cli.c

```diff
@@ -2106,40 +2500,62 @@ static int ssl_parse_server_psk_hint( mbedtls_ssl_context *ssl,
 
 #if defined(MBEDTLS_KEY_EXCHANGE_RSA_ENABLED) ||                           \
     defined(MBEDTLS_KEY_EXCHANGE_RSA_PSK_ENABLED)
-/*
- * Generate a pre-master secret and encrypt it with the server's RSA key
```

Comment:
Is this comment irrelevant now?

Reply:
It's no longer accurate. I re-introduced an adapted version.